lib/terraform/runner.rb

```diff
@@ -119,6 +103,19 @@ def json_post_arguments(payload)
         return JSON.generate(payload), "Content-Type" => "application/json".freeze
       end
 
+      def to_cloud_providers_conn_params(credentials)
```

Comment:
how about `provider_connection_parameters`... we don't really care that they're clouds, just that the we'll ask the instance of the class to build the connection parameters.